gettsim/data/pension_data.yaml

```diff
@@ -0,0 +1,537 @@
+alpha:
```

Comment:
Filename should be `ges_renten_vers`, seemed uncontroversial in the [current proposal](https://github.com/iza-institute-of-labor-economics/gettsim/issues/79#issuecomment-554283492) in #79.